File: app/src/main/java/users/profile/kotlinapi/Model/Shows.java
package users.profile.kotlinapi.Model;

import java.util.ArrayList;

public class Shows {
    private ArrayList<Show> listShow;

    public Shows() {
        this.listShow = new ArrayList<>();
    }
    public Boolean addShow(Show show){
        this.listShow.add(show);
        return true;
    }

}

File: app/src/main/java/users/profile/kotlinapi/Model/Shows.kt
package users.profile.kotlinapi.Model

class Shows() {
   lateinit  var  listShow:ArrayList<Show>

    init {
        listShow= ArrayList<Show>()
    }
   fun ajouter(show: Show){
       listShow.add(show)
   }
   @JvmName("getListShow1")
   fun getListShow():ArrayList<Show>{
       return  listShow
   }

    override fun toString(): String {
        var msg=""
        for (i in listShow){
            msg+="\n"+i.toString()
        }
        return msg
    }
}